spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRequiredDistributionAndOrdering.java

```diff
@@ -268,6 +268,23 @@ public void testDefaultSortOnLongTruncatedColumn() {
     assertEquals("Rows must match", expected, sql("SELECT * FROM %s ORDER BY c1", tableName));
   }
 
+  @Test
+  public void testDefaultSortOnBinaryColumn() {
```

Comment:
@jzhuge I added back this UT, not sure if is this what you expected.

Reply:
Yes, it is the unit test. However, what do you think if we move it from TestRequiredDistributionAndOrdering.java to TestIcebergExpressions.java? I feel the later test source file is a more place to add the unit test.

Reply:
Hi @jzhuge, I checked the `TestRequiredDistributionAndOrdering ` again, it seems the new UT would be more reasonable to put here. You could see the around UTs all are about partition and ordering tests. I renamed the UT name from `testDefaultSortOnBinaryColumn ` to `testDefaultSortOnBinaryBucketedColumn` to keep the name consistent with others. However, I could move it if you stand by your point.

Reply:
No, it is ok, I don't feel strongly about the move.

Good to go!